content/docs/user-guide/components/reference/physx/collider.md

```diff
@@ -116,35 +115,35 @@ Capsule resize mode has two linear manipulators. The manipulator at the top of t
 
 ![PhysX Collider component mode capsule resize manipulator](/images/user-guide/components/reference/physx/physx-collider-resize-capsule.png)
 
-### Resize (Physics Asset Shape)
-
-Physics Asset resize mode has a three-axis scale manipulator.
-
-![PhysX Collider component mode physics asset resize manipulator](/images/user-guide/components/reference/physx/physx-collider-resize-physicsasset.png)
-
-### Offset
+### Offset (Physics Asset Shape)
```

Comment:
Offset mode is the same for all shape types. Since the order has changed, suggest moving this to the top of the section and removing (Physics Asset Shape) from the heading.

```suggestion
### Offset
```